plugins/kubectl-neat.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: neat
+spec:
+  description: "If you try to `kubectl get` resources you have created, they will
```

Comment:
Is there a particular reason for not using multi-line yaml strings like
```yaml
description: |
  first line
  second line

  fourth line
```

Reply:
yes, the reason it that the plugin yaml file is auto generated from a json template. Since it's generated there's no way to preserve yaml features such as text blocks, but the result is functionally similar

Reply:
I see. In that case, that's fine. I don't want to break your pipeline :)
But could you make sure that the line length stays at around 80 characters? Long lines produce bad formatting on small terminals, so we try to avoid that.

Reply:
sure, I'll do that

Reply:
I managed to preserve text block style. also shortened to 80 chars per line